scio-coders/src/main/scala/com/spotify/scio/coders/Coder.scala

```diff
@@ -45,15 +47,23 @@ sealed trait Coder[T] extends Serializable
 final case class Beam[T] private (beam: BCoder[T]) extends Coder[T]
```

Comment:
`private` doesn't do anything there because it hides only `new`, but not `apply`